packages/dds/tree/src/shared-tree/treeApi.ts

```diff
@@ -66,6 +66,20 @@ export interface TreeApi extends TreeNodeApi {
 		tree: TView,
 		transaction: (root: TView["root"]) => void | "rollback",
 	): void;
+
+	/**
+	 * Check if the subtree defined by `parent`	contains `child`.
+	 *
+	 * @returns true if and only if the subtree rooted at `parent` contains `child`.
+	 * @remarks
+	 * Parent is considered to contain itself, so the case where `parent === child` returns true.
+	 *
+	 * This is handy when checking if moving parent into child would create a cycle and thus is invalid.
+	 *
+	 * This check walks the parents of child looking for parent,
```

Comment:
It wasn't clear to me (until I got to the bottom of this doc comment) that this is talking about _descendants_, not merely _direct_ children. I think that will be cleared up if you replace each place you said "child" with "descendant", and "parent" with "ancestor". Or, you could change the explanation to talk in terms of nodes and subtrees, e.g. "checks if nodeB is in the subtree rooted at nodeA".